draft-ietf-quic-transport.md

```diff
@@ -2560,11 +2594,16 @@ properly continue the connection.  An endpoint that wishes to communicate a
 fatal connection error MUST use a closing frame if it has sufficient state to do
 so.
 
-To support this process, a token is sent by endpoints.  The token is carried in
-the NEW_CONNECTION_ID frame sent by either peer, and servers can specify the
-stateless_reset_token transport parameter during the handshake (clients cannot
-because their transport parameters don't have confidentiality protection).  This
-value is protected by encryption, so only client and server know this value.
+To support this process, a token is sent by the endpoints that can be recovered
+or regenerated by the endpoint based on the information available in each packet
+even if the connection state was lost. This provides a protect mechanism to
+identify Stateless Reset packets from an attacker.
```

Comment:
"protect mechanism" is missing something, but I'm not sure what.  It doesn't permit identifying SRs generated by an attacker, because they'll just look like random noise.  I'd drop this last sentence, though your expansion on our assumptions about the token might be useful.

Reply:
What about: "This mechanism protects against forged Stateless Retry packets that could otherwise enable an off-path attacker to close a connection." Any better?

Reply:
That works.